db-cleanup/airflow-db-cleanup.py

```diff
@@ -1,6 +1,15 @@
+from airflow import version
+from pkg_resources import parse_version
+
+if parse_version(version.version).base_version == '1.10.5':
+   from airflow import settings
+elif parse_version(version.version).base_version == '1.10.3':
+   from airflow.models import settings
+else:
+   from airflow import settings
```

Comment:
You can just do: `from airflow import settings`, because it will work in 1.10.3 and 1.10.5. Previous version of import was not correct, check this out: https://github.com/apache/airflow/blob/1.10.3/airflow/models/__init__.py#L79